### ISSUE TITLE
Error while loading backend api

### ISSUE THREAD

**Issue by LuigiElleBalotta** [LearnWebCode/react-course#2]
Hello, I have an error while trying to start the backend api:

https://pastebin.com/mebf3dpQ

This is the "complete log":
https://pastebin.com/XjpFwWRq

I followed all your instruction!
Also I have internet connection working, so I don't know why it's going in timeout..

**Comment by SamuelOjes** [comment]
Make sure you check properly, your MONGODB connection string in your .env file to see if you entered your password, email and database  name correctly

**Comment by LuigiElleBalotta** [comment]
Yes, the configuration is correct. I added to the end of the string &ssl=true and it's working. 
Also in db.js there should be an output to let the user know that the server is running, a console.log at least because sometime, since there is no output, it seems that the connection to db goes in timeout

**Comment by SamuelOjes** [comment]
Great. Did you add it to the end of the mongodb connection String?. Today i tried running the backend server again, i am having the same issue right now.

**Comment by LuigiElleBalotta** [comment]
Yes at the end!
And put a console log inside the db connection function inside db.js for making sure that the connection was successful

**Comment by SamuelOjes** [comment]
Thanks. Think it has to do with some isp and mongodb err. Still having the issue, it worked yesterday though

**Comment by SamuelOjes** [comment]
Thanks it  working already, i used a different mongodb connection string, from mongodb and selected Nodejs Version 2.22.

**Comment by mojtabamaslahatbin** [comment]
i have same problem to start the backend api :(

**Comment by mojtabamaslahatbin** [comment]
@LuigiElleBalotta @SamuelOjes 
i solved this problem by allowing node.js to communicate through windows defender firewall.
go to this address in control panel :
" Control Panel >> Windows Defender Firewall >> allow an app or feature communicate through windows defender firewall"
and check both private and public checkbox for "node.js:Server-side JavaScript" and run your command again.

**Comment by Zainab98Azzam** [comment]
Hey guys, I had the same problem and I followed what @LuigiElleBalotta said, I added &ssl=true at the end of the string and console logged hi, and it's working! thank you so much you saved my day.
